projects/strongswan/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+./autogen.sh
+
+./configure CFLAGS="$CFLAGS -DNO_CHECK_MEMWIPE" LDFLAGS='--coverage' --enable-fuzzing --enable-coverage --with-libfuzzer=$LIB_FUZZING_ENGINE --enable-monolithic --disable-shared --enable-static
```

Comment:
what is LDFLAGS='--coverage' and --enable-coverage? 
If that's the coverage that libFuzzer or AFL uses, then CFLAGS should already have it set correctly. 
If that's a different form of coverage, do you really need it? 
(it may just stay there eating CPU cycles)